test/ui/testAppActions.js

```diff
@@ -942,4 +942,51 @@ describe('appActions', function() {
 
   });
 
+  describe('acknowledge', function() {
+    it('should record acknoledgment for user for specific key', function() {
+      var userid = 'foobar';
+      var key = 'daylightSavings';
+      app.state.targetId = userid;
+      appActions.acknowledge(key);
+      expect(app.state.acknowledgments[key][userid]).to.be.ok;
+    });
+  });
+
+  describe('shouldDisplayDaylightSavingsMessage', function() {
+    it('should return false when currentTime is less than time chnage time and user not seen message', function() {
+      var currentTimeIso = '2015-10-30T14:34:00';
+      app.state.targetTimezone = 'US/Pacific';
+      app.state.targetId = 'foobar';
+      app.state.acknowledgments = {};
+      app.state.acknowledgments.daylightSavings = {};
+      expect(appActions.shouldDisplayDaylightSavingsMessage(currentTimeIso)).to.be.false;
+    });
+
+    it('should return false when currentTime is less than time change time and user has seen message', function() {
+      var currentTimeIso = '2015-10-01T14:34:00';
+      app.state.targetTimezone = 'US/Pacific';
+      app.state.targetId = 'foobar';
+      app.state.acknowledgments = {};
+      app.state.acknowledgments.daylightSavings = { foobar: 134577844};
```

Comment:
So the stored timestamp is a Unix timestamp in seconds, not the milliseconds. Which are we storing? Please store milliseconds and update the test to be consistent - I get we aren't doing anything with this yet, but it's confusing...